indico/modules/rb_new/client/js/components/modals/BlockingModal.jsx

```diff
@@ -0,0 +1,253 @@
+                                            users/groups you specify in the "Allowed users/groups" list will be able
+                                            to create bookings for the specified rooms in the given timeframe.
+                                            You can also block rooms you do not own - however, those blockings have to
+                                            be approved by the owners of those rooms.
+                                        </Translate>
+                                    </Message.Content>
+                                </Message>
+                                <Message color="red" icon>
+                                    <Icon name="warning sign" />
+                                    <Message.Content>
+                                        <Translate>
+                                            When blocking rooms nobody but you, the rooms' managers and those
+                                            users/groups you specify in the "Allowed users/groups" list will be able to
+                                            create bookings for the specified rooms in the given timeframe. You can
+                                            also block rooms you do not own - however, those blockings have to be
+                                            approved by the owners of those rooms.
+                                        </Translate>
+                                    </Message.Content>
+                                </Message>
+                                <Divider hidden section />
+                                <Field name="rooms"
+                                       component={this.renderRoomSearchField}
+                                       disabled={submitting || submitSucceeded} />
+                            </Grid.Column>
+                            <Grid.Column width={8}>
+                                <Field name="allowed"
+                                       component={this.renderPrincipalSearchField}
+                                       disabled={submitting || submitSucceeded} />
+                                <Field name="period"
+                                       component={this.renderBlockingPeriodField}
+                                       disabled={submitting || submitSucceeded} />
+                                <Field name="reason"
+                                       label={Translate.string('Reason')}
+                                       component={ReduxFormField}
+                                       as={Form.TextArea}
+                                       format={formatters.trim}
+                                       placeholder={Translate.string('Reason for blocking')}
+                                       disabled={submitting || submitSucceeded}
+                                       formatOnBlur
+                                       required />
+                                {submitSucceeded && (
```

Comment:
I think you can use `<Message success>` here which will only show up when the form has `success` set.